mandos-go/json/integrationTests/example.scen.json

```diff
@@ -55,12 +55,14 @@
                             "ESDTRoleNFTBurn"
                         ]
                     },
+                    "username": "str:myusername.elrond",
                     "storage": {},
                     "code": ""
                 },
                 "address:smart_contract_address": {
```

Comment:
Maybe use `sc:` ?

Reply:
`sc:` is for smart contract addresses, this is the username

Reply:
I meant `address:smart_contract_address` could use `sc:`.